app/Dockerfile

```diff
@@ -1,7 +1,28 @@
 # Docker file for Node UI
 
 FROM node:16-alpine3.11
+
+ENV PORT 3001
```

Comment:
`ENV` is used to set environment variables in the container. But syntax should be `ENV PORT=3001` if this variable is being used in the app?

Reply:
@chrisaballard this is how it needed to be configured according to this tutorial https://dev.to/kumareth/next-js-docker-made-easy-2bok
Same with the other two questions you had about the volumes.